label_propagation.gql

```diff
@@ -0,0 +1,6 @@
+lp_twitter = loadgraph("$GRAPE_TEST/twitter.yaml")
+lp = loadapp("$GRAPE_TEST/labelpropagation.yaml")
```

Comment:
labelpropagation.yaml should be label_propagation.yaml

Reply:
Thanks, I'll fix that.